tests/test_incremental.py

```diff
@@ -122,7 +123,7 @@ def test_score_ndarrays():
     inc = Incremental(sgd, scoring='accuracy')
 
     inc.partial_fit(X, y, classes=[0, 1])
-    inc.fit(X, y)
+    inc.fit(X, y, classes=[0, 1])
```

Comment:
need this, as `classes_` no longer leasks through the `partial_fit` to the `fit` (a good thing!)